frontend/fingertips-frontend/lib/tableHelpers/mocks.ts

```diff
@@ -151,3 +151,98 @@ export const GROUPED_YEAR_DATA = {
     Female: [MOCK_HEALTH_DATA[1].healthData[2]],
   },
 };
+
+export const MOCK_HEALTH_DATA_WITH_TRENDS = [
+  {
+    areaCode: '1472',
+    areaName: 'Manchester - Counties and Unitary Authorities',
+    healthData: [
+      {
+        year: 2019,
+        count: 1400,
+        value: 222.305692,
+        lowerCi: 220.69151,
+        upperCi: 224.32766,
+        ageBand: 'All',
+        sex: 'All',
```

Comment:
Could all these be changed to `Persons` instead of `All` as the api no longer returns the `All` value for the sex field
```suggestion
        sex: 'Persons',
```

Reply:
Thanks - good point. I've updated. Please check you're happy with the changes.